tests/test_grammar.py

```diff
@@ -363,7 +364,8 @@ def test_single_element_no_interpolation(
         visitor = grammar_visitor.GrammarVisitor(
             node_interpolation_callback=None,  # type: ignore
             resolver_interpolation_callback=None,  # type: ignore
-            quoted_string_callback=lambda s: s,
+            quoted_string_callback=lambda s, memo: s,
+            memo=set(),
```

Comment:
Probably that `memo` could be `None` here

Reply:
Yup.
I only looked for set() in under the omegaconf module so I missed that one.